src/werkzeug/routing.py

```diff
@@ -631,8 +640,15 @@ def foo_with_slug(adapter, id):
         used to provide a match rule for the whole host.  This also means
         that the subdomain feature is disabled.
 
+    `websocket`
+        If True (defaults to False) this Rule represents a WebSocket,
```

Comment:
s/Rule/rule - i don't think we need to explicitly reference the class name here; similar places above don't do so either